addons/l10n_ma/i18n/l10n_ma.pot

```diff
@@ -663,3 +663,8 @@ msgstr ""
 #: model:account.report.line,name:l10n_ma.tax_report_part_b_14_nd
 msgid "Without right to deduct"
 msgstr ""
+
+#. module: l10n_ma
+#: model:ir.model.fields,help:l10n_ma.field_res_company__ice
+msgid "Common Company Identifier"
+msgstr ""
```

Comment:
useless since you're targeting master ;-) (all translations will be generated when freezing the master branch into a stable version)

Reply:
But not for localisation though :eyes: So i think we should have the pot and po file